release-content/0.15/release-notes/14920_Add_cached_run_system_API.md

```diff
@@ -1,4 +1,28 @@
-<!-- Add cached `run_system` API -->
-<!-- https://github.com/bevyengine/bevy/pull/14920 -->
+# Cached one-shot systems
 
-<!-- TODO -->
+**Bevy 0.15** introduces a new "cached" API for running one-shot systems:
+
+```rust
+// Run a system without manually registering it first:
+commands.run_system_cached(foo);
+commands.run_system_cached_with(bar, 42);
+
+// Reuse a registered system automatically:
```

Comment:
I don't think talking about system registration is useful.

Reply:
As in, don't mention `register_system_cached` up here? I know `register_system_cached` will be used less frequently than `run_system_cached`, but I figured that showing how it gives the same ID when called multiple times was a good demonstration of what the API does differently.

Reply:
Ah, I see your point now. Right, so I think that this is best left as a demonstration of the technical details below.

Reply:
Moved it later and added contrast.